### PR TITLE
show node name with error msg for ipmi

### PR DESCRIPTION
**Pull request by xuweibj**
#4728 

For simulation_test,simulation_test1 , mgt=ipmi.
UT:
```
# rspconfig simulation_test,simulation_test1,f6u03 bootmode
simulation_test: Error: Unsupported command: rspconfig bootmode for ipmi management method
simulation_test1: Error: Unsupported command: rspconfig bootmode for ipmi management method
f6u03: BMC BootMode : Regular
```

#4578 
```
# rflash boston01,boston02 -l
boston01: Error: The option -l is not supported or invalid update file specified
boston02: Error: The option -l is not supported or invalid update file specified
```

### CODE REVIEW

**Comment by zet809** [comment]
hi, @whowutwut , will you pls take a look at this PR? Thx!

File: xCAT-server/lib/xcat/plugins/ipmi.pm [hunk 1]
```diff
@@ -8312,26 +8329,39 @@ sub preprocess_request {
...
                         return;
                     }
                     $optset = 1;
                     $option = $1;
                 } else {
                     if ($optset eq 1) {
-                        $callback->({ errorcode => [1], data => [ "Usage Error: Cannot display and change attributes on the same command."] });
+                        foreach (@$all_noderange) {
+                            $error_data .= "\n" if ($error_data);
+                            $error_data .= "$_: Usage Error: Cannot display and change attributes on the same command for ipmi management method.";
```
Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/ipmi.pm]: Same comment above, this should really be a variable if we are duplicating these error messages so we can easily change the message once for each occurrence we have.

**Comment by xuweibj** [comment]
Hi @whowutwut , thank you for your comments. I have modified, please help to review again.

**Comment by zet809** [comment]
Let me merge this PR and apply new one if needed.

**Pull request merged**